src/commands/mod.rs

```diff
@@ -247,6 +247,19 @@ pub fn execute() -> Result<()> {
             .join(" ")
     );
 
+    #[cfg(feature = "profiling")]
+    let _guard = sentry::init((
+        env!("SENTRY_DSN").to_string(),
+        sentry::ClientOptions {
+            release: sentry::release_name!(),
+            user_agent: crate::constants::USER_AGENT.into(),
+            traces_sample_rate: 0.05,
```

Comment:
I wonder if its worth it to move `traces_sample_rate` and `profiles_sample_rate` to `constants.rs`